src/components/content/order/create/OrderSubmit.tsx

```diff
@@ -86,6 +93,20 @@ function OrderSubmit(state: OrderSubmitProps): React.JSX.Element {
         .concat('&latestVersion=', state.version)
         .concat('&billingMode=', state.billingMode);
 
+    const retryRequest = () => {
+        if (submitDeploymentRequest.isSuccess && submitDeploymentRequest.data.length > 0) {
+            redeployFailedDeploymentQuery.mutate(submitDeploymentRequest.data);
+        }
+    };
+
+    if (redeployFailedDeploymentQuery.isPending) {
+        void getServiceDetailsByIdQuery.refetch();
```

Comment:
i think this is still wrong. We must refetch only if the redeployFailedDeploymentQuery.isSuccess